src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const totalCosts = days * 40;
```

Comment:
Consider defining a constant for the daily rental rate (40) to improve code readability and maintainability.